src/SourceBuild/content/repo-projects/scenario-tests.proj

```diff
@@ -9,21 +9,19 @@
 
     <!-- Don't use the updated NuGet.config file that includes the live package feeds when testing source-only as in that configuration
          the Microsoft built packages should be used. -->
-    <TestArgs Condition="'$(DotNetBuildSourceOnly)' == 'true'">$(TestArgs) /p:TestRestoreConfigFile=$(OriginalNuGetConfigFile)</TestArgs>
+    <TestArgs Condition="'$(DotNetBuildSourceOnly)' == 'true' and '$(DotNetSourceOnlyTestOnly)' != 'true'">$(TestArgs) /p:TestRestoreConfigFile=$(OriginalNuGetConfigFile)</TestArgs>
+    <UseBootstrapArcade>true</UseBootstrapArcade>
```

Comment:
It's weird to have this un-conditionally set to true but then later for non-source-only builds, reference the arcade repo. Maybe add a condition?

Reply:
Fixed with https://github.com/dotnet/sdk/pull/48327/commits/483c97557f03602420f652b1946a295f6d733803